doc/fileexplorer.rst

```diff
@@ -1,28 +1,79 @@
-:orphan:
+#####
+Files 
+#####
 
-#############
-File Explorer
-#############
-
-The **File Explorer** pane is a built-in filesystem and directory browser allowing you to view and filter files and their properties, open them with the :doc:`editor` or an external tool, and perform common management tasks (varies by platform).
+The **Files** pane is a built-in filesystem and directory browser that allows you to view and filter files according to their type and extension.
+You can also open files with the :doc:`editor` or an external tool, and perform some operations.
 
 .. image:: images/file_explorer/file_explorer_standard.png
    :alt: Spyder File Explorer panel, showing a tree view of files and metadata
 
 |
 
 
-==========================
-Performing file operations
-==========================
 
-You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and (on Windows) opening an external file explorer.
+===============
+File Operations
+===============
+
+To browse between the files in your system, you can use the arrows in the top of the pane.
+You can expand/contract the folders displayed in the pane to deploy the files and subfolders hierarchically. 
+Double cliking a folder will open it showing the files on it and changing your working directory.
```

Comment:
```suggestion
Double-clicking a folder will open it, showing the files inside and making it your working directory.
```

Fix spelling, punctuation, wording/grammar and explain working dir change more explicitly.